src/app/app.module.ts

```diff
@@ -135,4 +136,4 @@ if (environment.production) {
   entryComponents: [],
   bootstrap: [AppComponent]
 })
-export class AppModule { }
+export class AppModule { }
```

Comment:
I believe tslint should show this for you, but please keep one newline at end of the file.